src/pages/SignIn.js

```diff
@@ -4,13 +4,23 @@ import { Clickable, HeroWrapper, Link } from 'src/components/common'
 import Modal from 'src/components/Modal'
 import SignInButton from 'src/components/SignInButton'
 import colors from 'src/libs/colors'
-import { isBioDataCatalyst, isFirecloud } from 'src/libs/config'
+import { isAnvil, isBioDataCatalyst, isFirecloud } from 'src/libs/config'
 import { getAppName } from 'src/libs/logos'
 import * as Nav from 'src/libs/nav'
 import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
+const styles = {
+  warningNoticeContainer: {
+    lineHeight: 1.5, marginTop: '2rem',
+    paddingTop: '1rem', borderTop: Style.standardLine
+  },
+  warningNotice: {
```

Comment:
I wanted to use this on l.77 too, but wasn't sure how to partially apply the style since only part of that warning notice is uppercase

Reply:
The way to do that would be to separate it out into two children within the div, and apply the styles to that one div. I am fine with punting on that though. (You would need to futz with the parent div so that the children appeared inline, which you could do by making it a flex container)